spec/chitter_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'chitter_manager'
+
+describe ChitterManager do
+
+  describe '.all' do
+    it 'returns a list of all messages' do
+      connection = PG.connect(dbname: 'chitter_test')
+      ChitterManager.post(message: 'This is my first peep')
+
+      peeps = ChitterManager.all
+
+      expect(peeps[0]).to be_a ChitterManager
```

Comment:
To make the test stronger, you could also check that the array only contains one item.